Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -267,14 +267,24 @@ void MemWatchWidget::onMemWatchContextMenuRequested(const QPoint& pos)
     contextMenu->addSeparator();
   }
 
+  MemWatchEntry* const entry{MemWatchModel::getEntryFromIndex(index)};
+  if (!entry || node->isGroup()) {
```

Comment:
It'd probably be a bit more sensible to test `node` and not mix it in `entry`:
```cpp
if (!node || node->isGroup()) {
```
And the other line could be reverted back to its original place.